libcodechecker/analyze/analyzers/analyzer_types.py

```diff
@@ -249,6 +314,7 @@ def __build_clangsa_config_handler(args, context):
         cmdline_checkers = None
 
     initialize_checkers(config_handler,
+                        context,
```

Comment:
I would prefer not to forward the whole context here. Context is a quite big object and it can be hard to write easily testable code if context is an argument of the function.
There are two attributes used from the context in the `initialize_checkers` method which could be passed here as an argument.